integration_tests/test_ica_precompile.py

```diff
@@ -302,3 +302,5 @@ def submit_msgs_ro(func, str):
     assert expected_seq == last_seq
     assert status == Status.FAIL
     assert cli_host.balance(ica_address, denom=denom) == balance
+
+    print(tcontract.events.OnPacketResult().get_logs())
```

Comment:
TODO, assert the logs

Reply:
yea, we might need get_event_data from the logs